build.gradle

```diff
@@ -350,6 +350,17 @@ ext.pomAttributes = {
   }
 }
 
+def getAllDependentProjects(project) {
+  if(project.configurations.hasProperty("runtime")) {
```

Comment:
is this necessary? if the subproject uses the Java plugin, then it should always have the `runtime` configuration